fdbclient/DatabaseBackupAgent.actor.cpp

```diff
@@ -2685,71 +2685,25 @@ class DatabaseBackupAgentImpl {
 		return Void();
 	}
 
-	ACTOR static Future<Void> atomicSwitchover(DatabaseBackupAgent* backupAgent,
-	                                           Database dest,
-	                                           Key tagName,
-	                                           Standalone<VectorRef<KeyRangeRef>> backupRanges,
-	                                           Key addPrefix,
-	                                           Key removePrefix,
-	                                           ForceAction forceAction,
-	                                           bool doSwitch,
-	                                           bool doLockDabatabse) {
-		state DatabaseBackupAgent drAgent(dest);
-		state UID destlogUid = wait(backupAgent->getLogUid(dest, tagName));
-		state EBackupState status = wait(backupAgent->getStateValue(dest, destlogUid));
-
-		TraceEvent("DBA_SwitchoverStart").detail("Status", status);
-		if (status != EBackupState::STATE_RUNNING_DIFFERENTIAL && status != EBackupState::STATE_COMPLETED) {
-			throw backup_duplicate();
-		}
-
-		if (!g_network->isSimulated() && !forceAction) {
-			state StatusObject srcStatus = wait(StatusClient::statusFetcher(backupAgent->taskBucket->src));
-			StatusObject destStatus = wait(StatusClient::statusFetcher(dest));
-			checkAtomicSwitchOverConfig(srcStatus, destStatus, tagName);
-		}
-
-		state UID logUid = deterministicRandom()->randomUniqueID();
-		state Key logUidValue = BinaryWriter::toValue(logUid, Unversioned());
-		state UID logUidCurrent = wait(drAgent.getLogUid(backupAgent->taskBucket->src, tagName));
-
-		if (logUidCurrent.isValid()) {
-			logUid = logUidCurrent;
-			logUidValue = BinaryWriter::toValue(logUid, Unversioned());
-		}
-
-		// Lock src, record commit version
-		state Transaction tr(backupAgent->taskBucket->src);
-		state Version commitVersion;
-		loop {
-			try {
-				if (doLockDabatabse) {
-					wait(lockDatabase(&tr, logUid));
-				}
-				tr.set(backupAgent->tagNames.pack(tagName), logUidValue);
-				wait(tr.commit());
-				commitVersion = tr.getCommittedVersion();
-				break;
-			} catch (Error& e) {
-				wait(tr.onError(e));
-			}
-		}
-
-		TraceEvent("DBA_SwitchoverLocked").detail("Version", commitVersion);
-
+	ACTOR static Future<Void> flushBackup(DatabaseBackupAgent* sourceBackupAgent,
```

Comment:
Optional change for consistency with other actors:

```suggestion
	ACTOR static Future<Void> flushBackup(DatabaseBackupAgent* backupAgent,
```